it/base/IntegrationTestBase.scala

```diff
@@ -1,66 +1,23 @@
 package base
 
-import controllers.actions.{FakeIdentifierAction, FakeIdentifierActionProvider, IdentifierAction, IdentifierActionProvider}
-import org.scalatest.Assertion
-import org.scalatest.concurrent.ScalaFutures
+import com.typesafe.config.ConfigFactory
 import org.scalatest.time.{Millis, Seconds, Span}
-import play.api.inject.bind
 import play.api.inject.guice.GuiceApplicationBuilder
-import play.api.mvc.ControllerComponents
-import play.api.test.Helpers.stubControllerComponents
-import play.api.{Application, Play}
-import play.modules.reactivemongo.ReactiveMongoApi
-import reactivemongo.play.json.collection.JSONCollection
-import uk.gov.hmrc.auth.core.AffinityGroup.Agent
+import play.api.{Application, Configuration}
+import uk.gov.hmrc.mongo.test.MongoSupport
 
-import scala.concurrent.ExecutionContext.Implicits.global
-import scala.concurrent.duration.Duration
-import scala.concurrent.{Await, Future}
-
-trait IntegrationTestBase extends ScalaFutures {
+trait IntegrationTestBase extends MongoSupport {
```

Comment:
this class is not needed at all you can use MongoSupport class on your test class straight and initialise repository there without the app at all, if you need help with this refactor let me know